app/main.py

```diff
@@ -1 +1,45 @@
-#  write your code here
+class Animal:
+
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> str:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = not self.is_hungry
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super(Cat, self).__init__(name, appetite=3, is_hungry=is_hungry)
+
+    @staticmethod
```

Comment:
Why is this method static?

Reply:
the catch_mouse() and bring_slippers() method does not use any attributes or instance methods of the Cat and Dog class. Therefore, in theory, these methods can be declared as static.

Reply:
A static method is a method that belongs to a class rather than an instance of a class. But, this method describes behaviour of an instance of the class. So, this method can't be static

Reply:
I understand, thank you. I will fix the code now

Reply:
please check my code